bndtools.core/src/org/bndtools/core/resolve/ui/ResolutionSuccessPanel.java

```diff
@@ -75,21 +76,19 @@ public ResolutionSuccessPanel(BndEditModel model, ResolutionResultPresenter pres
         this.model = model;
         this.presenter = presenter;
     }
-
+    
     public void createControl(Composite parent) {
         FormToolkit toolkit = new FormToolkit(parent.getDisplay());
-        composite = toolkit.createComposite(parent);
-
-        GridLayout layout = new GridLayout(1, false);
-        composite.setLayout(layout);
-
-        SashForm form = new SashForm(composite, SWT.VERTICAL);
```

Comment:
We can't remove the sash form, this is what gives users ability to grab the border between the sections and manually expand/contract them.  So can you reintroduce the sash form into the hierarchy?